CHANGELOG/CHANGELOG-2.1.md

```diff
@@ -0,0 +1,72 @@
+# Changelog 2.1
+
+## 2.1.0 - 2024-07-26
+
+### Added
+
+- Add a new feature `BGPPolicy` that allows users to initialize BGP process on selected Kubernetes Nodes and advertise Service IPs, Pod IPs, and Egress IPs to remote BGP peers.
+  - Add `BGPPolicy` API and Controller. ([#6009](https://github.com/antrea-io/antrea/pull/6009) [#6203](https://github.com/antrea-io/antrea/pull/6203), [@hongliangl])
+  - Add BGP datapath interface and implement goBGP integration. ([#6447](https://github.com/antrea-io/antrea/pull/6447), [@hongliangl])
+  - Add documentation for the `BGPPolicy` feature. ([#6524](https://github.com/antrea-io/antrea/pull/6524), [@hongliangl])
+  - Refer to [this document](https://github.com/antrea-io/antrea/blob/release-2.1/docs/bgp-policy.md) for more information about this feature.
+- Add a new feature `NodeLatencyMonitor` that allow users to do east/west connectivity monitoring and provides a latency query API.
+  - Add `NodeLatencyMonitor` feature in Antrea Agent that performs ping mesh and monitor the latency between Nodes. ([#6120](https://github.com/antrea-io/antrea/pull/6120), [@IRONICBo])
+  - Export Node latency query API definition. ([#6392](https://github.com/antrea-io/antrea/pull/6392), [@IRONICBo])
+- Add two new antctl commands to validate a K8s Cluster before Antrea installation, and allow users to validate basic network functionality after Antrea is installed.
+  - Add `antctl check cluster` command to to ensure that a K8s cluster is configured properly before Antrea installation. ([#6278](https://github.com/antrea-io/antrea/pull/6278), [@kanha-gupta])
+  - Add `antctl check installation` command to conduct Pod/Service connectivity checks and verify basic NetworkPolicy rules. ([#6133](https://github.com/antrea-io/antrea/pull/6133) [#6313](https://github.com/antrea-io/antrea/pull/6313) [#6367](https://github.com/antrea-io/antrea/pull/6367), [@kanha-gupta])
+  - Add documentation for the new `antctl check` commands. ([#6373](https://github.com/antrea-io/antrea/pull/6373), [@kanha-gupta])
+  - Refer to [this document](https://github.com/antrea-io/antrea/blob/release-2.1/docs/antctl.md#performing-checks-to-facilitate-installation-process) for more information about these new commands.
+
+### Changed
+
+- Ensure Antrea Proxy handles all Service traffic with proxyAll enabled, even when kube-proxy is present. ([#6308](https://github.com/antrea-io/antrea/pull/6308), [@hongliangl])
+- Optimize the containerized OVS installation on Windows. ([#6383](https://github.com/antrea-io/antrea/pull/6383), [@wenyingd])
```

Comment:
maybe add: manual installation of the OVS kernel driver is usually not required anymore

as that is relevant to users

Reply:
Update